template_parts/Bids_modality_agnostic/run1.json

```diff
@@ -1,70 +1,70 @@
 {
-    "elabftw": {
-        "extra_fields_groups": [
-            {
-                "id": 1,
-                "name": "Run1"
-            }
-        ]
+  "elabftw": {
+    "display_main_text": true,
+    "extra_fields_groups": [
+      {
+        "id": 2,
+        "name": "Run01 "
+      }
+    ]
+  },
+  "extra_fields": {
+
+    "RunNumber01": {
+      "type": "text",
+      "value": "",
+      "group_id": 2,
+      "position": 0
     },
-    "extra_fields": {
-        "EventsSet1": {
-            "type": "text",
-            "value": "",
-            "group_id": 1,
-            "position": 4,
-            "description": "Details about the events associated with this set."
-        },
-        "RunNumber1": {
-            "type": "text",
-            "value": "",
-            "group_id": 1,
-            "position": 0
-        },
-        "StartTime1": {
-            "type": "text",
-            "value": "",
-            "group_id": 1,
-            "position": 5
-        },
-        "RunComment1": {
-            "type": "text",
-            "value": "",
-            "group_id": 1,
-            "position": 6,
-            "description": "Comment about the subject behavior during the run."
-        },
-        "Acquisition1": {
-            "type": "text",
-            "value": "",
-            "group_id": 1,
-            "position": 3,
-            "description": "Details about the acquisition process."
-        },
-        "DataFilePath1": {
-            "type": "text",
-            "value": "",
-            "group_id": 1,
-            "description": "the path where the Protocole 's file is saved"
-        },
-        "ImageFilePath1": {
-            "type": "text",
-            "value": "",
-            "group_id": 1,
-            "description": "This field will store the name of an image file or a link to an image"
-        },
-        "AnalyseFilePath1": {
-            "type": "text",
-            "value": "",
-            "group_id": 1,
-            "description": "This field will store the name of the analysis fil"
-        },
-        "TaskDescription1": {
-            "type": "text",
-            "value": "",
-            "group_id": 1,
-            "position": 2,
-            "description": "Description of the task"
-        }
+    "TaskName01": {
+      "type": "text",
+      "value": "",
+      "group_id": 2,
+      "position": 1,
+      "required": true,
+      "description": "Code name of the task (no space and only alphanumeric characters).\n Ex: 'rest or facesnback or headnodding'"
+    },
+    "TaskDescription01": {
+      "type": "text",
+      "value": "",
+      "group_id": 2,
+      "position": 2,
+      "description": "Description of the task"
+    },
+    "Acquisition01": {
+      "type": "text",
+      "value": "",
+      "group_id": 2,
+      "position": 3,
+      "description": "Details about the acquisition process."
```

Comment:
I think this field is aimed as being used in BIDS, like TaskName... So here also, it should be asked to have "(no space and only alphanumeric characters)"... Can you check in the BIDS specs and add this?

Reply:
if we want to use it to get a value for the "acquisition" BIDS entity (https://bids-specification.readthedocs.io/en/stable/glossary.html#acquisition-entities), we need that there is no space and etc., so that it can be used in the filename...

to be checked later...